### ISSUE TITLE
Show more than 20 items in GearBar

### ISSUE THREAD

**Issue by rparliment** [RagedUnicorn/wow-classic-gearmenu#64]
I carry more then 20 different trinkets around when doing PvP, when highlighting the trinket slots in the GearBar it will only show 20 of the trinkets, is there a way to have this number expanded to adjust for the number of available trinkets to equip? I have 23 trinkets currently and have my eyes set on 4 more for the current phase.

Also would be nice to have the ability to left or right click a trinket to determine which slot it is queued into, instead of requiring two different slots.

**Comment by RagedUnicorn** [comment]
Hi @rparliment 

I increased the amount of trinkets to 40. I also had a look at implementing "combined equipping". I did implement a form of it. Not sure though if it is exactly what you had in mind. I tinkered with two possible scenarios on how to implement this.

Scenario 1:
In a way assigning one of the trinketslots to left and the other to right. Meaning if I click left on an item it would land in the "left slot" and if I click right it lands in the right slot.

Scenario 2:
If I click left on an item it would normally equip the item in the currently hovered slot. If I click right however it would be equipped in the opposite slot than the one that I currently hover.

I decided to implement Scenario 2. The reasoning for me was that I can't really see how saving a slot would help. I can't imagine that I would hide one of the trinket slots. I still need to see what item I have equipped and what the status of that item is. Additionally I found it nonintuitive to use. The slots are not exactly categorizable into left and right. Depending on where the player has his trinketslots the whole left and right makes no sense.

Still interested in your or anybody else's feedback if someone tried the feature.

Commit reference: https://github.com/RagedUnicorn/wow-classic-gearmenu/commit/d87e15ed2cccc8dd775cef00bc395b9fc0e8549e

Note: This feature will be in the next release.

Regards,
RagedUnicorn
